src/pal/src/loader/module.cpp

```diff
@@ -815,8 +815,7 @@ Return value:
     non-NULL - the base address of the mapped image
     NULL - error, with last error set.
 --*/
-void *
-PALAPI
+PVOID
```

Comment:
Why drop PALAPI?

Reply:
In `pal.h`, there is:

```h
PALIMPORT
PVOID
PALAPI
PAL_LOADLoadPEFile(HANDLE hFile);
```

Should the implementation have both `PALIMPORT` and `PALAPI` or just the latter?

Reply:
Just the latter - it is how all other similar implementations look like as far as I can tell.